blueocean-core-js/src/js/fetch.js

```diff
@@ -4,8 +4,29 @@ import isoFetch from 'isomorphic-fetch';
 import utils from './utils';
 import config from './config';
 
+let refreshToken = null;
 export const FetchFunctions = {
-        /**
+    checkRefreshHeader(response) {
+        const _refreshToken = response.headers.get('X-Blueocean-Refresher');
```

Comment:
that's a bit of an odd name for a header - wouldn't normally use a verb in a header.